package.json

```diff
@@ -79,6 +79,7 @@
 		"express": "^4.18.1",
 		"helmet": "^6.0.0",
 		"morgan": "^1.10.0",
+		"navigate": "^0.3.6",
```

Comment:
I suspect you haven't fully rebased on top of the main branch: navigate is already in package.json

Reply:
Woops, got it wrong. You don't need navigate as you are not using it: you already have react-router-dom

Reply:
fixed.
i downloaded it while i use testing the navigate